components/webgpu/lib.rs

```diff
@@ -1088,7 +1110,7 @@ impl<'a> WGPU<'a> {
                         data,
                     } => {
                         let global = &self.global;
-                        gfx_select!(queue_id => global.queue_write_buffer(
+                        let _ = gfx_select!(queue_id => global.queue_write_buffer(
```

Comment:
TODO: handle OOM here?

Reply:
Yep, I'll add it. We only report errors for `Createxxx` operations at the moment.